### DISCUSSION TITLE
Failed to create thumbnail - short Hoffman data

### DISCUSSION THREAD

**Discussion by FlyingPersian** [photoprism/photoprism#1982] [Q&A] [answered]
Hello,

I have 33 pictures in my database that PP fails to create a thumbnail for. The error says:

```
index: failed creating thumbnails for 'Footing 09-12.JPG' (invalid JPEG format: short Huffman data)

invalid JPEG format: short Huffman data
```

I can open the pictures on other devices, but not via PP. They fully load and seem okay.  I have attached one, maybe it helps.

![Footing 09-12](https://user-images.githubusercontent.com/8209643/151091346-ada50798-323e-47f9-9487-b2b2080e5619.JPG)

I'm running PP on a RPI 4b+ in docker. I'm using the latest version.

**Comment by lastzero** [comment, marked as answer]
The file is broken, some JPEG decoders decode the file until the position where it's broken (grey area):

![photo_2022-01-26_10-47-35](https://user-images.githubusercontent.com/301686/151140605-643aa337-4ea0-44a1-90c9-f918ddb1c396.jpg)

The Go JPEG implementation (and others) don't do this. It is definitely not a "bug", rather an additional feature of some decoders. You may find tools online to repair broken files.

**Reply by lastzero** [reply]
see https://github.com/golang/go/issues/10447

**Reply by lastzero** [reply]
Explicitly added this to our Troubleshooting Checklist for *Missing Pictures*:

https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures
